### PR TITLE
Win warnings

### PR DESCRIPTION
**Pull request by RussTedrake**
silence some compiler warnings on windows.

### CODE REVIEW

File: systems/plants/RigidBody.cpp [hunk 1]
```diff
@@ -38,7 +38,7 @@ RigidBody::RigidBody(void) :
 	body_index = 0;
 	mass = 0.0;
 	floating = 0;
-	pitch = 0.0;
```
Comment (hongkai-dai) [inline systems/plants/RigidBody.cpp]: Why from double to float? Also need to fix the indentation

Comment (RussTedrake) [inline systems/plants/RigidBody.cpp]: to int.  pitch is an int.

Comment (RussTedrake) [inline systems/plants/RigidBody.cpp]: indentation looks ok to me?  (perhaps it's your browser?)

**Comment by hongkai-dai** [comment]
+1

**Pull request merged**